src/components/SiteHeader.js

```diff
@@ -22,32 +27,26 @@ const LinkedLogo = React.memo(() =>
                 alt={CUSTOM_HEADER || "Bento"}
             />
         </div>
-    </Link>,
-);
-
+    </Link>
+));
 
-const CustomHeaderText = React.memo(() =>
-    <h1 style={{ color: "rgba(255, 255, 255, 0.95)", float: "left", margin: "0 24px 0 0" }}>
-        {CUSTOM_HEADER}
-    </h1>,
-);
+const CustomHeaderText = React.memo(() => (
+    <h1 style={{ color: "rgba(255, 255, 255, 0.95)", float: "left", margin: "0 24px 0 0" }}>{CUSTOM_HEADER}</h1>
```

Comment:
can you turn off your auto-formatter that formats files with rules we don't enforce in bento_web? it makes it hard to review this code.

Reply:
This one is an outcome of the printWidth property, which it doesnt violate but prettier uses to reformat to best fit code